lib/subscribers/mysql/pool-namespace-query.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const MySQLPoolQuerySubscriber = require('./pool-query.js')
```

Comment:
I _think_ the namespace files are related to pool cluster correct? I would rename these to `pool-cluster-`